src/osd/SnapMapper.cc

```diff
@@ -116,8 +116,8 @@ int OSDriver::get_next(
   CRIMSON_DEBUG("OSDriver::{} key {}", __func__, key);
   using crimson::os::FuturizedStore;
   return interruptor::green_get(os->omap_get_values(
-    ch, hoid, key
-  ).safe_then_unpack([&key, next] (bool, FuturizedStore::Shard::omap_values_t&& vals) {
+    ch, hoid, FuturizedStore::Shard::omap_keys_t{key}
+  ).safe_then([&key, next] (FuturizedStore::Shard::omap_values_t&& vals) {
```

Comment:
Does this change belong to this commit?

Reply:
it is introduced by dropping omap_get_values(, , &start), so change it to use omap_get_values( , , keyset).